src/mediarithmics/api/plugin/emailtemplaterenderer/EmailInterface.ts

```diff
@@ -1,10 +1,10 @@
 export interface PluginEmailMeta {
-  from_email: string;
-  from_name: string;
-  to_email: string;
-  to_name: string;
-  reply_to: string;
-  subject_line: string;
+  from_email?: string;
```

Comment:
Is it a backward compatible change ?

Reply:
Turning something mandatory into something optional is always backwards compatible, isn't it?